x-pack/metricbeat/module/gcp/gke/_meta/fields.yml

```diff
@@ -0,0 +1,214 @@
+- name: gke
+  type: group
+  release: beta
+  fields:
+  - name: container
+    type: group
+    fields:
+    - name: cpu
+      type: group
+      fields:
+      - name: core_usage_time.value
+        type: double
+        description: Cumulative CPU usage on all cores used by the container in seconds.
+          Sampled every 60 seconds.
+      - name: limit_cores.value
+        type: double
+        description: CPU cores limit of the container. Sampled every 60 seconds.
+      - name: limit_utilization.value
+        type: double
+        description: The fraction of the CPU limit that is currently in use on the
+          instance. This value cannot exceed 1 as usage cannot exceed the limit. Sampled
+          every 60 seconds. After sampling, data is not visible for up to 240 seconds.
+      - name: request_cores.value
+        type: double
+        description: Number of CPU cores requested by the container. Sampled every
+          60 seconds. After sampling, data is not visible for up to 120 seconds.
+      - name: request_utilization.value
+        type: double
+        description: The fraction of the requested CPU that is currently in use on
+          the instance. This value can be greater than 1 as usage can exceed the request.
+          Sampled every 60 seconds. After sampling, data is not visible for up to
+          240 seconds.
+    - name: ephemeral_storage
+      type: group
+      fields:
+      - name: limit_bytes.value
+        type: long
+        description: Local ephemeral storage limit in bytes. Sampled every 60 seconds.
+      - name: request_bytes.value
+        type: long
+        description: Local ephemeral storage request in bytes. Sampled every 60 seconds.
+      - name: used_bytes.value
+        type: long
+        description: Local ephemeral storage usage in bytes. Sampled every 60 seconds.
+    - name: memory
+      type: group
+      fields:
+      - name: limit_bytes.value
+        type: long
+        description: Memory limit of the container in bytes. Sampled every 60 seconds.
+      - name: limit_utilization.value
+        type: double
+        description: The fraction of the memory limit that is currently in use on
```

Comment:
I think we used `pct` for percentages. See:

```
  - name: memory.broker.pct
    description: The percentage of the memory limit used.
    type: scaled_float
    format: percent
```

Sometimes you need an additional Javascript processor to convert to the right format:

```
processors:
  - script:
      lang: javascript
      source: >
        function process(event) {
          var broker_memory_broker_pct = event.Get("activemq.broker.memory.broker.pct")
          if (broker_memory_broker_pct != null) {
            event.Put("activemq.broker.memory.broker.pct", broker_memory_broker_pct / 100.0)
          }

          var broker_memory_temp_pct = event.Get("activemq.broker.memory.temp.pct")
          if (broker_memory_temp_pct != null) {
            event.Put("activemq.broker.memory.temp.pct", broker_memory_temp_pct / 100.0)
          }

          var broker_memory_store_pct = event.Get("activemq.broker.memory.store.pct")
          if (broker_memory_store_pct != null) {
            event.Put("activemq.broker.memory.store.pct", broker_memory_store_pct / 100.0)
          }
        }
```

(based on the ActiveMQ module.